### ISSUE TITLE
Released Datenstrom Yellow 0.8.4

### ISSUE THREAD

**Issue by markseu** [datenstrom/yellow#422]
What's new:

- better static site generator
- added progress indicator to static site generator
- added support for strikethrough and task list
- added accessibility improvements for screenreader, thanks Steffen
- updated languages, replaced EditVersion with EditAbout
- updated API, removed backwards compatibility with old API
- renamed extension Googlemaps to Googlemap

Notes: Backwards compatibility with the old API has been removed. Some extensions may not be compatible with this release and may not work. If your website is affected, please contact the developer and let him/her know.

Do you have questions about this release? Let us know. 😄

**Comment by i-do** [comment]
[Mark](@markseu), can you give me the right direction to API manual. [Old one](https://github.com/datenstrom/yellow/issues/401#issuecomment-467388196) doesn't work any more.

**Comment by markseu** [comment]
You're right. Just replace EditVersion with EditAbout, for example: `EditAboutTitle: Check updates`.

**Comment by BsNoSi** [comment]
I investigated my *bundle-challenge* that still exists.

At first I thought I simply have to set the correct `ResouceLocatione` that I missed before.

This lead to this css (resonable partes):

```css
/* orignial css file expanded */
@font-face {
	font-family: 'Bitter';
	font-style: normal;
	font-weight: 400;
	src: url(Bitter-regular.woff2)  format('woff2'), url(Bitter-regular.woff) format('woff');
}
@font-face {
	font-family: 'Bitter';
	font-style: bold;
	font-weight: bold;
	src: url(Bitter-700.woff2) format('woff2'), url(Bitter-700.woff) format('woff');
}
@font-face {
	font-family: 'Bitter';
	font-style: italic;
	font-weight: 400;
	src: url(Bitter-italic.woff2) format('woff2'), url(Bitter-italic.woff) format('woff');
}
@font-face {
	font-family: 'Inconsolata';
	font-style: normal;
	src: url(Inconsolata-regular.woff2) format('woff2'), url(Inconsolata-regular.woff) format('woff');
}
/* bundle.php result expanded */
@font-face {
	font-family: 'Bitter';
	font-style: normal;
	font-weight: 400;
	src: url(/system/resources/Bitter-regular.woff2) format('woff2'), url(/system/resources/Bitter-regular.woff) format('woff')
}
@font-face {
	font-family: 'Bitter';
	font-style: bold;
	font-weight: 700;
	src: url(/system/resources/Bitter-700.woff2) format('woff2'), url(/system/resources/Bitter-700.woff) format('woff')
}
@font-face {
	font-family: 'Bitter';
	font-style: italic;
	font-weight: 400;
	src: url(/system/resources/Bitter-italic.woff2) format('woff2'), url(/system/resources/Bitter-italic.woff) format('woff')
}
@font-face {
	font-family: 'Inconsolata';
	font-style: normal;
	src: url(/system/resources/Inconsolata-regular.woff2) format('woff2'), url(/system/resources/Inconsolata-regular.woff) format('woff')
}
```

To me this looks identical except the fact that `bundle.php` adds the full path.

Bundle.php css delivers this page:

![image](https://user-images.githubusercontent.com/2784683/56958585-eda0c880-6b4a-11e9-8869-1059d1c0eb1e.png)


Removing `bundle.php` from the extensions I receive this (which I expect):

![image](https://user-images.githubusercontent.com/2784683/56958593-f4c7d680-6b4a-11e9-9548-1e6c18599718.png)


I have to admit that I do not have a glimpse what happens here. Investigating the delivered pages rise no hints (to me)  but the bold font is used if I use `bundle.php`. 

At the moment I will leave out `bundle.php` but I'd prefer understanding what happens there.

**Comment by markseu** [comment]
I think the bundle extension was not changed this release. Please open a new issue.

**Comment by BsNoSi** [comment]
> I investigated my _bundle-challenge_ that still exists. […]

Thanks to Mark this is fixed.

I investigated this a long time but could not find the error. Mark offered his help and checked a demo website I created for him. He found the mistake in a place I would never expect it as a reason (and therefore never even would take a look on): the CSS file had "UTF-8 BOM" coding. Removing the "BOM" solved it. Wow.

**A VERY BIG THANK YOU to Mark** hoping this did not took to much of your time.
